src/patterns/components/toggle/base.hbs

```diff
@@ -13,7 +13,7 @@ hasAngularCodeInfo: true
   </a>
 
   <div data-sprk-toggle="content">
-    <p class="sprk-b-TypeBodyFour sprk-u-pts">
+    <p class="sprk-b-TypeBodyFour sprk-u-pts sprk-u-pbs">
```

Comment:
Toggles need this extra padding because of the font with our type collision stuff.